### PR TITLE
Release 12.0.0

### PR DESCRIPTION
**Pull request by r0mant**
Do not merge yet.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,616 @@
 # Changelog
 
+## 12.0.0
+
+Teleport 12 brings the following marquee features and improvements:
+
+- Device Trust (Preview, Enterprise only)
+- Non-AD support for Desktop Access (Preview, Enterprise only)
```
Comment (zmb3) [inline CHANGELOG.md:8]:
```suggestion
- Passwordless Windows access for local users (Preview, Enterprise only)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,616 @@
...
+- Signed/notarized macOS binaries
+
+### Device Trust (Preview, Enterprise only)
+
+Teleport 12 includes a preview of our upcoming Device Trust feature, which
+allows administrators to require that Teleport access is performed from an
+authenticated and trusted device.
+
+This preview release requires macOS and a native client like tsh or Teleport
+Connect. These native leverage the Secure Enclave on macOS to solve device
```
Comment (zmb3) [inline CHANGELOG.md:28]:
```suggestion
Connect. These clients leverage the Secure Enclave on macOS to solve device
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,616 @@
...
+The teleport-cluster Helm chart underwent significant changes in Teleport 12. To
+upgrade from an older version of the Helm chart deployed in “custom” mode, use
+the following migration guide:
+
+https://goteleport.com/docs/ver/12.x/deploy-a-cluster/helm-deployments/migration-v12/
+
+Additionally, PSPs are removed from the chart when installing on Kubernetes 1.23
+and higher to account for the deprecation/removal of PSPs by Kubernetes.
+
+#### Tctl auth export
```
Comment (zmb3) [inline CHANGELOG.md:103]:
```suggestion
#### tctl auth export
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,616 @@
...
+
+The tctl auth export command only exports the private key when passing the
+--keys flag. Previously it would output the certificate and private key
+together.
+
+#### Desktop Access
+
+Windows Desktop sessions disable the wallpaper by default, improving
+performance. To restore the previous behavior, add show_desktop_wallpaper: true
```
Comment (zmb3) [inline CHANGELOG.md:112]:
```suggestion
performance. To restore the previous behavior, add `show_desktop_wallpaper: true`
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,616 @@
 # Changelog
 
+## 12.0.0
+
+Teleport 12 brings the following marquee features and improvements:
+
+- Device Trust (Preview, Enterprise only)
+- Passwordless Windows access for local users (Preview, Enterprise only)
```
Comment (alexfornuto) [inline CHANGELOG.md:8]: Sorry to be a pedant, but we could access local users previously, what's changed is that we don't need integration with Active Directory to do it.

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,616 @@
...
+- Per-pod RBAC for Kubernetes Access
+- Azure and GCP CLI support for Application Access
+- Support for more databases in Database Access
+- AWS DynamoDB
+- AWS Redshift Serverless
+- AWS RDS Proxy for PostgreSQL/MySQL
+- Azure SQLServer Auto Discovery
+- Azure Flexible Servers
```
Comment (alexfornuto) [inline CHANGELOG.md:16]: Should these be indented bullets under the previous point?

File: CHANGELOG.md (hunk 5 above)
Comment (alexfornuto) [inline CHANGELOG.md:8]: Sorry to be a pedant, but we could access local users previously, what's changed is that we don't need integration with Active Directory to do it.

File: CHANGELOG.md (hunk 6 above)
Comment (alexfornuto) [inline CHANGELOG.md:16]: Should these be indented bullets under the previous point?

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,616 @@
...
+### Per-pod RBAC for Kubernetes Access
+Teleport 12 extends RBAC to support controlling access to individual pods in
+Kubernetes clusters. Pod RBAC integrates with existing Teleport RBAC features
+such as role templating and access requests.
+
+### Azure and GCP CLI support for Application Access
+
+In Teleport 12 administrators can interact with Azure and GCP APIs through
+Application Access using “tsh az” and “tsh gcloud” CLI commands, or using
+standard “az” and “gcloud” tools through the local application proxy.
```
Comment (alexfornuto) [inline CHANGELOG.md:50]: Should we use inline code notation for CLI tools?

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,5 +1,616 @@
...
+### Refactored Helm charts
+
+The “teleport-cluster” Helm chart underwent significant refactoring in Teleport
+12 to provide better scalability and UX. Proxy and Auth are now separate
+deployments and the new “scratch” chart mode makes it easier to provide a custom
+Teleport config.
+
+“Custom” mode users should follow the migration guide:
+
+https://goteleport.com/docs/ver/12.x/deploy-a-cluster/helm-deployments/migration-v12/
```
Comment (alexfornuto) [inline CHANGELOG.md:70]: Note here to remove the `ver/12.x/` string once v12 is the default Docs version.

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,5 +1,616 @@
...
+Please familiarize yourself with the following potentially disruptive changes in
+Teleport 12 before upgrading.
+
+#### Helm charts
+
+The teleport-cluster Helm chart underwent significant changes in Teleport 12. To
+upgrade from an older version of the Helm chart deployed in “custom” mode, use
+the following migration guide:
+
+https://goteleport.com/docs/ver/12.x/deploy-a-cluster/helm-deployments/migration-v12/
```
Comment (alexfornuto) [inline CHANGELOG.md:98]: Note here to remove the ver/12.x/ string once v12 is the default Docs version.

**Review by russjones — APPROVED** [review]

**Review by alexfornuto — APPROVED** [review]
👍 to not block the release, but noted some potential changes.

File: CHANGELOG.md (hunk 5 above)
Comment (zmb3) [reply CHANGELOG.md:8]: - This is the approved language from marketing that we'll be using for the release. "Non AD" doesn't mean as much to new customers who don't know that we only supported AD to start with.
- No, we could never access local users previously, only AD enrolled users.

**Review by zmb3 — APPROVED** [review]

File: CHANGELOG.md (hunk 6 above)
Comment (r0mant) [reply CHANGELOG.md:16]: Nope, this is a separate "feature".

Comment (r0mant) [reply CHANGELOG.md:16]: Actually, I misunderstood your comment. Yes, they should be, I'll update.

File: CHANGELOG.md (hunk 7 above)
Comment (r0mant) [reply CHANGELOG.md:50]: Sure.

File: CHANGELOG.md (hunk 1 above)
Comment (alexfornuto) [reply CHANGELOG.md:8]: I think a regression happened here.

Comment (r0mant) [reply CHANGELOG.md:8]: Hmm yeah not sure what happened, I might have force-pushed without pulling locally. I'll update these again.

**Pull request merged**